### DISCUSSION TITLE
How to turn this off?

### DISCUSSION THREAD

**Discussion by PatrickChodowski** [aevyrie/bevy_mod_picking#220] [Q&A] [answered]
I am aware of possible stupidity of this question but I have honestly no idea (I am hoping for one of those moments when I find the solution immediately after asking someone else) 

But... how to turn off this input debugging?

![bevy_mod_picking](https://github.com/aevyrie/bevy_mod_picking/assets/11600009/a5b1a173-1e2b-4af1-812e-b62d4efc74f1)

I added the .add_plugins(DefaultPickingPlugins) to main app with no other configuration. I tried selecting features in cargo.toml: 

bevy_mod_picking = {version = "0.13", features = ["highlight","selection"]}

and after rebuilding the app it didnt change anything. 

Is there an example with how to configure logging and debugging options ? Ideally I would like to hide this Mouse input and suppress logging in the console: 
![image](https://github.com/aevyrie/bevy_mod_picking/assets/11600009/bcbb9853-8bb7-4af8-84d8-dfdcd649aeb3)

Thank you for help! I am happy to contribute some examples of using the crate once I understand the 0.13 version better

**Comment by aevyrie** [comment, marked as answer]
This is the debug plugin feature: https://github.com/aevyrie/bevy_mod_picking/blob/1f7df34a30a6bdfd4d0512bea6bc83e69ec98b11/Cargo.toml#L49

You can turn this off by either:

- Disabling the default features, and only pulling in the features you want. Doing it this way will mean you will not need to compile anything you don't use.
- Adding the `DefaultPickingPlugins`, then disabling `DebugPickingPlugin`:
```rs
.add_plugins(
    DefaultPickingPlugins.build()
        .disable::<DebugPickingPlugin>()
)
```

**Comment by PatrickChodowski** [comment]
Thank you for quick reply! Well, thats on my lack of knowledge about  cargo.toml. I thought the features= will be enough :(

**Reply by aevyrie** [reply]
Take a look at this: https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature

Hope that helps. 🙂
